File: java-code/src/classes/PayrollEmployee.java
package classes;

public class PayrollEmployee extends FullTimeEmployee{

  public PayrollEmployee(String firstName, String lastName, Position position, String dob, Department department,
                         Manager manager, Double salary) {
    super(firstName, lastName, position, dob, department, manager, salary);
  }

  // METHODS
  public void processPayroll(){
    System.out.println("Processing payroll");
  }

  public void reportToManager() {
    System.out.println(GetFirstName() + " Is driving down to go visit " + GetFullName());
  }
}

File: c#-code/src/classes/PayrollEmployee.cs
namespace APLCAssignment3.Classes
{
    using APLCAssignment3.Classes.Base; 

    public class PayrollEmployee : FullTimeEmployee
    {
        public PayrollEmployee(string firstName, string lastName, Position position, string dob, 
                               Department department, Manager manager, double salary)
            : base(firstName, lastName, position, dob, department, manager, salary)
        {}

        //methods
        public void ProcessPayroll()
        {
            Console.WriteLine("Processing payroll");
        }

        public override void ReportToManager()
        {
            Console.WriteLine($"{GetFirstName()} Is driving down to go visit {GetFullName()}");
        }
    }
}